### PR TITLE
Show wallpaper through when fullscreen

### PR DESCRIPTION
**Pull request by erikinkinen**

### CODE REVIEW

**Comment by erikinkinen** [comment]
Requires https://github.com/nemomobile-ux/glacier-home/pull/189

File: src/controls/qml/qmldir [hunk 1]
```diff
@@ -33,6 +33,7 @@ TextField 1.0 TextField.qml
 ProgressBar 1.0 ProgressBar.qml
 ActionButton 1.0 ActionButton.qml
 NemoIcon 1.0 NemoIcon.qml
+Wallpaper 1.0 Wallpaper.qml
```
Comment (neochapay) [inline src/controls/qml/qmldir:36]: no needed

File: src/controls/qml/Wallpaper.qml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+import QtQuick.Controls.Private 1.0
+import QtQuick.Controls.Nemo 1.0
+import QtQuick.Controls.Styles.Nemo 1.0
+
+Rectangle {
+    id: backgroundWallpaper
+    anchors.fill: parent
+    visible: width != Screen.width || height != Screen.height
+    color: Theme.backgroundColor
+}
```
Comment (neochapay) [inline src/controls/qml/Wallpaper.qml]: new line

File: src/styles/SliderStyle.qml [hunk 3]
```diff
@@ -27,7 +27,7 @@ SliderStyle{
     handle: Rectangle {
         id: handle
         anchors.centerIn: parent
-        color: Theme.backgroundColor
+        color: "transparent"
```
Comment (neochapay) [inline src/styles/SliderStyle.qml]: Maybe change from Rectangle to Item ?

File: src/styles/qml/ToolBarStyle.qml [hunk 4]
```diff
@@ -32,7 +32,7 @@ Style {
         implicitWidth: Theme.itemHeightSmall
         Rectangle {
             anchors.fill: parent
-            color: Theme.backgroundColor
+            color: "transparent"
```
Comment (neochapay) [inline src/styles/qml/ToolBarStyle.qml]: Maybe change from Rectangle to Item ?

File: src/controls/qml/qmldir (hunk 1 above)
Comment (erikinkinen) [reply src/controls/qml/qmldir:36]: file:///usr/lib/qt/qml/QtQuick/Controls/Nemo/ApplicationWindow.qml:192:5: Wallpaper is not a type

**Pull request merged**